### PR TITLE
[nasa-ghg] Add custom login page template and fix titles

### PR DESCRIPTION
**Pull request by j08lue**
- [x] Fix metadata for nasa-ghg hub
- [x] Point to customized login page template for the `staging` hub

Ready for review/merging.

### CODE REVIEW

File: config/clusters/nasa-ghg/common.values.yaml [hunk 1]
```diff
@@ -19,9 +19,9 @@ basehub:
       homepage:
         templateVars:
           org:
-            name: "NASA Green House Gases Center"
+            name: "U.S. Greenhouse Gas Center"
             logo_url: https://raw.githubusercontent.com/US-GHG-Center/ghgc-docs/87204d4a4e5f29c335a8f905c73de551aabd4845/Logo/ghg-logo.svg
```
Comment (slesaad) [inline config/clusters/nasa-ghg/common.values.yaml:23]: this is still the old logo, do we wanna replace it with the placeholder?

Comment (j08lue) [reply config/clusters/nasa-ghg/common.values.yaml:23]: Yes, please.

File: config/clusters/nasa-ghg/common.values.yaml [hunk 2]
```diff
@@ -19,9 +19,9 @@ basehub:
...
-            url: https://www.nasa.gov/emd/ghg
```
[the remark this answers is not in the document]
Comment (j08lue) [reply config/clusters/nasa-ghg/common.values.yaml:24]: if it cannot be empty, let us just link to https://www.earthdata.nasa.gov/esds for now and update later.

File: config/clusters/nasa-ghg/common.values.yaml (hunk 1 above)
Comment (j08lue) [inline config/clusters/nasa-ghg/common.values.yaml:23]:
```suggestion
            logo_url: https://raw.githubusercontent.com/US-GHG-Center/ghgc-docs/b818ba6fdd3c43ede04b110975bf39d248c40df6/Logo/ghg-logo.svg
```

**Review by slesaad — APPROVED** [review]

**Comment by j08lue** [comment]
Ready for your review, pls, @2i2c-org/engineering.

**Review by consideRatio — APPROVED** [review]

**Comment by slesaad** [comment]
well, this didn't work as expected. @consideRatio we were hoping the page would be updated with templates from here: https://github.com/US-GHG-Center/ghgc-hub-homepage ?

**Pull request merged**